pkg/ocm/share/repository/nextcloud/nextcloud.go

```diff
@@ -166,25 +168,40 @@ func (sm *Manager) StoreShare(ctx context.Context, share *ocm.Share) (*ocm.Share
 // GetShare gets the information for a share by the given ref.
 func (sm *Manager) GetShare(ctx context.Context, user *userpb.User, ref *ocm.ShareReference) (*ocm.Share, error) {
 	data, err := json.Marshal(ref)
+	log := appctx.GetLogger(ctx)
 	if err != nil {
 		return nil, err
 	}
-	_, body, err := sm.do(ctx, Action{"GetShare", string(data)}, getUsername(user))
+	_, body, err := sm.do(ctx, Action{"GetSentShareByToken", string(data)}, "nobody") // getUsername(user))
 	if err != nil {
 		return nil, err
 	}
 
-	altResult := &ShareAltMap{}
+	altResult := ShareAltMap{}
 	if err := json.Unmarshal(body, &altResult); err != nil {
 		return nil, err
 	}
+	rejson, err := json.Marshal(altResult)
```

Comment:
can you delete this? If you want to log the structure maybe you can consider of using the github.com/gdexlab/go-render lib